src/Microsoft.IdentityModel.JsonWebTokens/Results/TokenDecryptingResult.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Microsoft.IdentityModel.Tokens;
+
+#nullable enable
+namespace Microsoft.IdentityModel.JsonWebTokens
+{
+    /// <summary>
+    /// Contains the result of decrypting a JWT in clear text.
+    /// The <see cref="TokenValidationResult"/> contains a collection of <see cref="ValidationResult"/> for each step in the token validation.
+    /// </summary>
+    internal class TokenDecryptingResult : ValidationResult
```

Comment:
I think the name TokenDecryptionResult would better match the other ValidationResults.